libpod/container_internal.go

```diff
@@ -131,14 +133,92 @@ func (c *Container) AttachSocketPath() string {
 	return filepath.Join(c.runtime.ociRuntime.socketsDir, c.ID(), "attach")
 }
 
+// exitFilePath gets the path to the container's exit file
+func (c *Container) exitFilePath() string {
+	return filepath.Join(c.runtime.ociRuntime.exitsDir, c.ID())
+}
+
+// create a bundle path and associated files for an exec session
+func (c *Container) createExecBundle(sessionID string) (err error) {
+	bundlePath := c.execBundlePath(sessionID)
+	if createErr := os.MkdirAll(bundlePath, execDirPermission); createErr != nil {
+		return createErr
+	}
+	defer func() {
+		if err != nil {
+			if err2 := os.Remove(bundlePath); err != nil {
+				logrus.Warnf("error removing exec bundle after creation caused another error: %v", err2)
+			}
+		}
+	}()
+	f, err2 := os.OpenFile(c.execLogPath(sessionID), os.O_RDONLY|os.O_CREATE, execFilePermission)
```

Comment:
Do we need to create this? I thought conmon would create it for us

Reply:
I feel like I remember there being some error because conmon didn't...

Reply:
nvm it works